### ISSUE TITLE
Jhipster image in the homepage disappears after changing the theme

### ISSUE THREAD

**Issue by srinisubra** [deepu105/generator-jhipster-bootswatch#4]
Hi Deepu105, Jhipster image in the homepage disappears after changing the theme. 
css class .hipster in main.css that gets affected after theme change
.hipster {
    display: inline-block;
    width: 347px;
    height: 497px;
    background: url("../images/hipster.png") no-repeat center top;
    background-size: contain;
}

**Comment by deepu105** [comment]
What browser are you using? I couldn't replicate this in crome, is there
any console error?

**Comment by srinisubra** [comment]
Sorry could not get a chance to update this thread earlier. It is happening in both Firefox V. 43 and google chrome Version 47.0.2526.106 m. No console errors. The moment I changed my theme to default, image re appears. Strange thing is, development banner is not affected but the jhipster image during theme selection. 
How do I set the bootswatch theme by default, for instance Simplex ? Thanks in advance
